packages/actor-rdf-resolve-quad-pattern-rdfjs-source/lib/IRdfJsSourceExtended.ts

```diff
@@ -1,6 +1,18 @@
 import type * as RDF from '@rdfjs/types';
 
 export interface IRdfJsSourceExtended extends RDF.Source {
+  /**
+   * A record indicating supported features of this source.
+   */
+  features?: {
+    /**
+     * If true, this source supports passing quad patterns with quoted quad patterns in the `match` method.
+     * If false (or if `features` is `undefined`), such quoted quad patterns can not be passed,
+     * and must be replaced by `undefined` and filtered by the caller afterwards.
```

Comment:
'filtered' here got me confused, I understood it as 'such quads would need to be removed by the caller afterwards'.

More clear to me would be:
If false (or if `features` is `undefined`), such quoted quad patterns can not be passed.
A new pattern can be derived, where they are replaced by `undefined`, and evaluated agains the store.
Some of the returned quads may not match agains the original pattern and need to be removed by the caller.

Reply:
A side thought.
This approach will work for the case when store supports storing quoted triples but does not supports matching agains them.

For the case when store does not support storing them (and matching agains, as this is a feature on top), all of the resulting quads, matched agains altered quad patterns will be non-matching agains the original pattern, which would add overhead for the store to resolve and return them and for the client to match and remove.
Perhaps to spare the overhead we could have an additional feature `quotedTripleStoring` or smth, and when it's set and `quotedTripleFiltering` is not set - simply remove quad pattern altogether?

Reply:
Also, is it 'quoted triples' or 'quoted quads'? (`quotedQuadsFiltering`?)